framework/xds_k8s_testcase.py

```diff
@@ -682,6 +682,84 @@ def assertAllBackendsReceivedRpcs(self, lb_stats):
                 msg=f"Backend {backend} did not receive a single RPC",
             )
 
+    def assertClientEventuallyReachesSteadyState(
+        self,
+        test_client: XdsTestClient,
+        rpc_type: str,
+        num_rpcs: int,
+        threshold_percent: int,
+        *,
+        retry_timeout: dt.timedelta = dt.timedelta(minutes=20),
+        retry_wait: dt.timedelta = dt.timedelta(seconds=2),
+        steady_state_delay: dt.timedelta = dt.timedelta(seconds=5),
```

Comment:
Can we use constants for those default values similar to `TD_CONFIG_MAX_WAIT`?

Reply:
I used `assertDrainingEndpointsCount` as an example here. I don't see any existing method that uses constants for default argument values.

Reply:
yea, I'm good with defining the defaults here without constants.